src/main/java/com/linkedin/kafka/clients/producer/LiKafkaProducerConfig.java

```diff
@@ -29,8 +32,11 @@
   public static final String AUDITOR_CLASS_CONFIG = "auditor.class";
   public static final String KEY_SERIALIZER_CLASS_CONFIG = ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG;
   public static final String VALUE_SERIALIZER_CLASS_CONFIG = ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG;
-  public static final String SEGMENT_SERIALIZER_CLASS_CONFIG = "segment.serializer";
   public static final String CURRENT_PRODUCER = "current.producer";
+  public static final String UUID_FACTORY_CLASS_CONFIG = "uuid.factory.class";
+  public static final String HEADER_PARSER_CONFIG = LiKafkaConsumerConfig.HEADER_PARSER_CONFIG;
+
+  public static final String HEADER_PARSER_DOC = LiKafkaConsumerConfig.HEADER_PARSER_DOC;
```

Comment:
Having the serializer code with the deserializer code makes it easier to compare them so you know the serializer and deserializer are working together correctly.  I'm sure why Kafka has them separated other than to make misconfiguration more likely.